web/static/httprest/connect/index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>autokitteh HTTP/REST Integration</title>
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="/static/apple-touch-icon.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="/static/favicon-32x32.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="/static/favicon-16x16.png"
+    />
+    <link rel="manifest" href="/static/site.webmanifest" />
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <meta name="robots" content="noindex, nofollow" />
+    <script src="form.js" defer></script>
+  </head>
+
+  <body>
+    <a href="/" target="_self">
+      <img class="banner" src="/static/banner.svg" alt="autokitteh" />
+    </a>
+
+    <table class="title">
+      <td><img class="logo" src="/static/images/http.svg" alt="Logo" /></td>
+      <td><h1>HTTP / REST - Create a New Connection</h1></td>
+    </table>
+
+    <div class="toggle-container">
+      <button class="toggle" id="toggletab1" onclick="toggleTab('tab1')">
+        No Auth
+      </button>
+      <span class="toggle-sep"></span>
+      <button class="toggle" id="toggletab2" onclick="toggleTab('tab2')">
+        Basic
+      </button>
+      <span class="toggle-sep"></span>
+      <button class="toggle" id="toggletab3" onclick="toggleTab('tab3')">
+        Bearer
+      </button>
+    </div>
+
+    <form method="post" action="/httprest/save">
```

Comment:
CSRF?

Reply:
This isn't really a CSRF attack vector because we don't trust the input coming from this form in any meaningful way.

We could add additional security, e.g. an "Origin" HTTP header, but:
1. This can be forged too
2. We don't particularly mind where the data is coming from
3. Any attack that can be carried out against the "/save" webhook can be carried against "index.html" as well